client/my-sites/exporter/export-card/index.jsx

```diff
@@ -77,7 +80,8 @@ class ExportCard extends Component {
 						onClickExport={ this.props.exportSelectedItems }
 					/>
 				</FoldableCard>
-				{ this.props.isExporting && <Interval onTick={ fetchStatus } period={ EVERY_SECOND } /> }
+				{ this.props.isExporting && ! isJetpackSite &&
+				<Interval onTick={ fetchStatus } period={ EVERY_SECOND } /> }
```

Comment:
If that's the only difference, I would suggest assuming that export is async, and immediately resolve the promise for the sync implementation.